totalRP3/Modules/Register/Main/RegisterExchange.lua

```diff
@@ -479,15 +479,9 @@ local function incomingInformationTypeSent(structure, senderID, channel)
 	QueryCooldowns[informationType][senderID] = nil;
 	TRP3_Addon:TriggerEvent("REGISTER_DATA_RECEIVED", senderID, informationType);
 
-	local decodedData = data;
-	-- If the data is a string, we assume that it was compressed.
-	if type(data) == "string" then
-		decodedData = Utils.serial.safeDecompressCodedStructure(decodedData, {});
-	end
```

Comment:
I'm 99.9% certain this code path can never be hit - the four information types that this is set up for always serialize data as a table, never as a string, and there's no calls to the utility that compresses data via LibCompress anyway.

I assume this was to support ancient versions of the addon, but it's definitely no longer needed at this point.